magento/magento2
# [MFTF] Refactoring AdminCheckDashboardWithChartsTest

### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Go to Admin Dashboard
2. Grab orders quantity
3. Place/invoice/ship an order
4. Reload the Dashboard
5. Check the display
6. Check if orders quantity increased



### Resolved issues:
1. [x] resolves magento/magento2#31523: [MFTF] Refactoring AdminCheckDashboardWithChartsTest
